browser/fragments/platform-nativescript.js

```diff
@@ -0,0 +1,71 @@
+require('nativescript-websockets');
+
+/**
+ * atob polyfill
+ * @param input
+ * @returns {string}
+ * Based on From: https://github.com/davidchambers/Base64.js/blob/master/base64.js
+ * License is Apache 2.0 also
+ */
+global.atob = function(input) {
```

Comment:
Ably-js already has a base64 decode in `browser/lib/util`. We should avoid duplication and choose a single version.